aten/src/ATen/xpu/xpu_functions.yaml

```diff
@@ -0,0 +1,55 @@
+# This yaml is used only for testing the XPU codegen functionality. 
```

Comment:
Why do we need to add this yaml to the source code? I think it should be the data file of test folder. Why is it not be a part of `test/xpu`?

Reply:
Indeed, have moved into test/xpu, thanks for the comment